web/modules/custom/dpl_opening_hours/src/tests/src/Kernel/OpeningHoursResourceTest.php

```diff
@@ -122,20 +124,23 @@ public function testList(): void {
    * Test that opening hours filters work.
    */
   public function testListFilters(): void {
-    $this->createOpeningHours(new DateTime("yesterday"), branchId: 1);
-    $this->createOpeningHours(new DateTime("now"), branchId: 1);
-    $this->createOpeningHours(new DateTime("tomorrow"), branchId: 1);
-    $this->createOpeningHours(new DateTime("tomorrow"), branchId: 2);
+    $now = new DateTime("now");
+    $tomorrow = new DateTime("tomorrow");
+    $yesterday = new DateTime("yesterday");
```

Comment:
Should be be replaced by something like:


```suggestion
    $now = new DateTime("now");
    $tomorrow = (clone $now)->modify("+1day");
    $yesterday = (clone $now)->modify("-1day");
```